src/relayserver/RelayServer.js

```diff
@@ -11,6 +11,13 @@ const StakeManagerABI = require('../common/interfaces/IStakeManager')
 const getDataToSign = require('../common/EIP712/Eip712Helper')
 const RelayRequest = require('../common/EIP712/RelayRequest')
 const utils = require('../common/utils')
+const Chains = {
+  mainnet: { chainId: 1, name: 'mainnet' },
```

Comment:
but `web3.eth.net.getNetworkType()` does it

Reply:
Ok ty